extras.lua

```diff
@@ -190,3 +190,31 @@ Simulationcraft.upgradeTable = {
   [531] = 2 -- 2/2 -> 10
 }
 
+Simulationcraft.slotIds = {
```

Comment:
I think the slot map is a little off. Tried linking a weapon and it maps it to a trinket slot in the simc output.

This is what I have in some Raidbots code for slotId -> simc slot name that I think maps to the `INV_` slot names:

```
  export const slotMap = {
    1: 'head',
    2: 'neck',
    3: 'shoulder',
    4: 'shirt',
    5: 'chest',
    6: 'waist',
    7: 'legs',
    8: 'feet',
    9: 'wrist',
    10: 'hands',
    11: 'finger',
    12: 'trinket',
    13: 'main_hand', // weapon
    14: 'off_hand', // shield
    15: 'main_hand',
    16: 'back',
    17: 'main_hand', // two-hand
    19: 'tabard',
    20: 'chest',
    21: 'main_hand',
    22: 'off_hand',
    23: 'off_hand',
    26: 'main_hand', // some ranged weapons
    28: 'off_hand',
  };
```

Reply:
The original INV_ to slot id table I had was copied off of wowwiki and untested - however, I have confirmed that it was correct. However, the simc addon does not use the slot ids, but instead uses its own numbering system. I've updated the table to have the correct values. These values are also different from those you posted - I do not know what those correspond to.